### PR TITLE
Swagger

### PR DESCRIPTION
**Pull request by mikolaslorant**
## Summary

* added swagger service to api (BookController) and model (Book)

## Trello card

https://trello.com/c/7lZzllmu/40-swagger

### CODE REVIEW

File: training/src/main/java/wolox/training/models/Book.java [hunk 1]
```diff
@@ -12,40 +14,51 @@
...
+    @ApiModelProperty(notes = "The year of publication: 2019, 2018, etc.")
     private String year;
 
     @Column(nullable = false)
+    @ApiModelProperty(notes = "The number of pages the book has.")
     private int pages;
 
     @Column(nullable = false)
+    @ApiModelProperty(notes = "The number of pages the book has.")
     private String isbn;
```
Comment (leandrocostawolox) [inline training/src/main/java/wolox/training/models/Book.java:58]: The api model property for isbn is not right, the isbn is the property used to identify any book (Like a unique document number, but for books)

**Review by leandrocostawolox — CHANGES_REQUESTED** [review]

File: training/src/main/java/wolox/training/models/Book.java (hunk 1 above)
Comment (mikolaslorant) [reply training/src/main/java/wolox/training/models/Book.java:58]: yes, sorry it was a copy mistake. My bad :(

Comment (mikolaslorant) [reply training/src/main/java/wolox/training/models/Book.java:58]: Fixed

Comment (leandrocostawolox) [reply training/src/main/java/wolox/training/models/Book.java:58]: Great! No problem :)

**Review by leandrocostawolox — APPROVED** [review]

**Review by IgnOliveto — APPROVED** [review]

**Pull request merged**